### PR TITLE
GLTFLoader: Flip normalScale without tangents.

### LINKED ISSUE #11438: GLTFLoader: Normal-Tangent Test model result is incorrect
##### Description of the problem 

I tried to display the [Normal-Tangent Test model](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/NormalTangentTest).
However, the displayed result seems to be different from Khronos' sample.

[Three.js + Normal-Tangent Test model result:](https://cdn.rawgit.com/cx20/gltf-test/4d14007e47668a3631cb5a50169b01445cb0aae6/examples/threejs/index.html?category=tutorialModels&model=NormalTangentTest&scale=1&type=glTF)
image

[Khronos sample loader + Normal-Tangent Test model result:](http://github.khronos.org/glTF-WebGL-PBR/)
image

I think that this sample model should have the same left and right results.

Related : https://emackey.github.io/testing-pbr/normal-tangent-readme.html

/cc @emackey 

##### Three.js version

- [x] Dev
- [ ] r85
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

##### Hardware Requirements (graphics card, VR Device, ...)

ThinkPad X260 + Windows 10 + Intel HD Graphics 520

### PR DESCRIPTION
**Pull request by elalish**
Fixes #11438 

After reading this [comment](https://github.com/mrdoob/three.js/issues/11438#issuecomment-545651421) I applied this change to model-viewer. It appears to be the right change, given all the models I've seen. I think it also makes sense given the chain rule explanation given in that thread as well.

### CODE REVIEW

**Comment by WestLangley** [comment]
> I think it also makes sense given the chain rule explanation given in that thread as well.

I assume you are referring to [this](https://github.com/mrdoob/three.js/issues/11438#issuecomment-507003995)?

**Comment by elalish** [comment]
@WestLangley Indeed, much appreciated!

File: examples/js/loaders/GLTFLoader.js [hunk 1]
```diff
@@ -2262,6 +2262,14 @@ THREE.GLTFLoader = ( function () {
 
 			if ( materialDef.name !== undefined ) material.name = materialDef.name;
 
+			// Normal map textures use OpenGL conventions:
+			// https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#materialnormaltexture
```
Comment (WestLangley) [inline examples/js/loaders/GLTFLoader.js]: I am not sure that comment explains the ensuing code block...

Maybe referencing https://github.com/mrdoob/three.js/issues/11438#issuecomment-507003995 would be OK?

File: examples/js/loaders/GLTFLoader.js [hunk 2]
```diff
@@ -2262,6 +2262,13 @@ THREE.GLTFLoader = ( function () {
 
 			if ( materialDef.name !== undefined ) material.name = materialDef.name;
 
+			// https://github.com/mrdoob/three.js/issues/11438#issuecomment-507003995
+			if ( material.normalScale && ! material.vertexTangents ) {
+
+				material.normalScale.y = - material.normalScale.y;
+
+			}
```
Comment (donmccurdy) [inline examples/js/loaders/GLTFLoader.js]: Can you check that this is OK on a model with `vertexTangents`? Note that we're in the `loadMaterial` method here, and `.vertexTangents` wouldn't be set until later, in the `assignFinalMaterial` method, when we have access to the mesh and can confirm whether it has tangents. I think the change will need to be moved there.

Comment (elalish) [reply examples/js/loaders/GLTFLoader.js]: Indeed, found the repro with your shoe.glb. Moved and verified the fix. Thanks!

**Review by donmccurdy — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**